### ISSUE TITLE
mdx-loader not working with react-scripts@5.0.0

### ISSUE THREAD

**Issue by thediveo** [frontarm/mdx-util#97]
After updating an existing React app [lxkns](https://github.com/thediveo/lxkns) to react-scripts 5.0.0 the documented method of using mdx-loader with babel-loader

```typescript
import AboutMDX from "!babel-loader!mdx-loader!./About.mdx"
```

does not work anymore. `AboutMDX` contains a string which is the path name to the Javascript file compiled from the .mdx source, but not the required React component or function.

**Comment by mzusev** [comment]
Yup, having the same issue here. Had to downgrade react-scripts to 4.0.3
